CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### v2.0.1
+
+* Support for the Distance Matrix has been added, this replaces the distance API.
+* The `RouteProgress` object is now immutable and structure has been split up.
+* Support for direction annotations has been added.
```

Comment:
Check consistency of lower/upper case, it's "directions" here, and "Directions" below.